src/hooks/Guilds/ens/useENSPublicResolverContract.ts

```diff
@@ -19,7 +16,7 @@ export function useENSAvatarUri(ensName: string, chainId?: number) {
     addressOrName: resolver?.address,
     contractInterface: ensPublicResolver.abi,
     functionName: 'text',
-    args: [convertToNameHash(ensName), 'avatar'],
+    args: [convertToNameHash(ensName).nameHash, 'avatar'],

```

Comment:
Since we have { error } from `convertToNameHash` shouldn't we use that to prevent this query to run with null nameHash? 
``` 
const { error, nameHash } = convertToNameHash(ensName); 
const { data, ...rest } = useContractRead({
  ...
  enabled: !error
 });
 ```

Reply:
Totally, with this we prevent a lot of calls to `useContractRead`.

Added!